connectivity/tests/errors.go

```diff
@@ -146,14 +209,13 @@ func (n *noErrorsInLogs) checkErrorsInLogs(logs string, t *check.Test) {
 		}
 	}
 	if len(uniqueFailures) > 0 {
-		failures := make([]string, 0, len(uniqueFailures))
+		var failures strings.Builder
 		for f, c := range uniqueFailures {
-			failures = append(failures, f)
-
-			t.Logf("Found %q in logs %d times\n", f, c)
+			failures.WriteRune('\n')
+			failures.WriteString(f)
+			failures.WriteString(fmt.Sprintf(" (%d occurrences)", c))
 		}
-		failureMsgs := strings.Join(failures, "\n")
-		t.Failf("Found %d logs matching list of errors that must be investigated:\n%s", len(uniqueFailures), failureMsgs)
+		a.Failf("Found %d logs in %s matching list of errors that must be investigated:%s", len(uniqueFailures), id, failures.String())
```

Comment:
q: do we actually match this anywhere, it appears like we just report on unique errors.

Reply:
Sorry, I'm not sure I'm fully understanding your question here. If you refer to the *matching list* wording, that refers to the fact that we check each log entry against a set of [known *unexpected* messages](https://github.com/giorio94/cilium-cli/blob/17bb0867880ca0d9b06efcaa9388b631a85e5c37/connectivity/tests/errors.go#L30-L52) (with possible exceptions), and trigger a failure if any of them is found. Multiple occurrences of the same entry are then coalescenced for the sake of brevity.